railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -342,6 +349,7 @@ func RunningAddresses(ctx context.Context, kubeCfg *rest.Config, publishService
 				addrs = append(addrs, ip.IP)
 			}
 		}
+
```

Comment:
nit: should usually avoid whitespace changes in areas you're not actually modifying.

Reply:
Revert :shrug: